.circleci/config.yml

```diff
@@ -0,0 +1,167 @@
+defaults: &defaults
+  docker:
+    - image: buildpack-deps:xenial
+  working_directory: ~/
+
+test-defaults: &test-defaults
+  <<: *defaults
+  steps:
+    - checkout:
+        path: emscripten/
+    - attach_workspace:
+        # Must be absolute path or relative path from working_directory
+        at: ~/
+
+    - run: |
+        apt-get update
```

Comment:
could these `apt-get, pip` things be done in the `build` stage once, instead of in each job?

Reply:
Yes if we share the whole `/usr/bin` directory, but I'm not sure we should do that.